.snyk/requirements.txt

```diff
@@ -17,3 +17,4 @@ pre-commit>=3.5.0
 pytest==8.1.1
 pytest-randomly==3.15.0
 syrupy==4.6.1
+hatchling
```

Comment:
Hatchling has MPL-2.0 license like certifi

Reply:
Then why we use it as requirement for building snowflake cli?